cc/tf_bt_utils.cc

```diff
@@ -112,6 +112,18 @@ void WriteGameExamples(const std::string& gcp_project_name,
 
   auto row_prefix = absl::StrFormat(kGameRowFormat, game_counter);
   WriteTfExamples(table, row_prefix, examples);
+  if (!player.options().resign_enabled) {
+    int bleakest_move = 0;
+    float bleakest_q = 0.0;
+    FindBleakestMove(player, &bleakest_move, &bleakest_q);
+    auto bleak_row_name =
+        absl::StrFormat(kPrefixAndMoveFormat, row_prefix, bleakest_move);
+    google::cloud::bigtable::SingleRowMutation row_mutation(bleak_row_name);
+    auto q_data = absl::StrFormat("%f", bleakest_q);
```

Comment:
I think you can just use `absl::StrCat` here instead of `absl::StrFormat`

Reply:
Yes, that's cleaner, and moreover, I see that `StrCat` will use 6 digits of *precision* (which I prefer in this case) rather than 6 digits to the right of the decimal, which is what the `"%f"` specifier gets me. Thanks!